__tests__/endpoints.test.js

```diff
@@ -32,15 +32,17 @@ describe("GET /api/articles/:article_id", () => {
       .get("/api/articles/1")
       .expect(200)
       .then(({ body: { article } }) => {
-        expect(article).toEqual({
-          article_id: 1,
-          title: "Living in the shadow of a great man",
-          author: "butter_bridge",
-          created_at: "2020-07-09T20:11:00.000Z",
-          topic: "mitch",
-          votes: 100,
-          body: "I find this existence challenging",
-        });
+        expect(article).toEqual(
+          expect.objectContaining({

```

Comment:
Nice refactor! This definitely avoids the issue of brittle tests as you add more :)

Reply:
Thanks very much :)